src/main/resources/liquibase/workbench_changelog/16_1_0.xml

```diff
@@ -15,5 +15,559 @@
 			DROP TABLE workbench_sidebar_category_link_role;
 		</sql>
 	</changeSet>
+	<changeSet author="diego" id="v16.1.0-2">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1"> select count(rp.role_id) > 0 from role_permission rp inner join permission p on rp.permission_id = p.permission_id where p.name = 'BREEDING_ACTIVITIES'</sqlCheck>
+        </preConditions>
+        <comment>Remove Breeding activities permission from role_permission table</comment>
+        <sql dbms="mysql" splitStatements="true">
+			set @permission_id = (SELECT permission_id FROM permission WHERE name ='MANAGE_GERMPLASM');
```

Comment:
I think will be easier to read doing: 
`set @breeding_activities_pid = (SELECT permission_id FROM permission WHERE name ='BREEDING_ACTIVITIES');
insert into role_permission(role_id,permission_id)
select rp.role_id, p.permission_id from role_permission rp
inner join permission p on rp.permission_id = p.parent_id
where rp.permission_id = @breeding_activities_pid;
delete from role_permission where permission_id = @breeding_activities_pid;
`